sled-agent/src/sled_agent.rs

```diff
@@ -152,8 +156,13 @@ struct SledAgentInner {
     // ID of the Sled
     id: Uuid,
 
-    // Sled underlay address
-    addr: SocketAddrV6,
+    // The sled's initial configuration
+    config: Config,
```

Comment:
We're storing the config here so we can see if we should be launching the "stub" flavor of the switch zone or not. There really isn't an issue with storing the rest of it here, so I'm keeping it for convenience.